domains/tp.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "My website!",
+  "repo": "https://github.com/techyporcupine/portfolio-site",
+  "owner": {
+    "username": "techyporcupine",
+    "email": "mail@cb-tech.me"
+  },
+  "record": {
+    "URL": "https://cb-tech.me"
+  },
```

```suggestion
  }
```
This comma should not be here.